### PR TITLE
EvaluatedFormula is a curve

### LINKED ISSUE #16: Create a dashboard for the new PLparameters.pdf
PLparameters.pdf has new predictions, we should show them in another dashboard. And perhaps show them also for the countries with old predictions as well, for comparison.

### PR DESCRIPTION
**Pull request by lukipuki**
Episode #8301 of Naming Is Hard

### CODE REVIEW

File: covid_graphs/covid_graphs/formula.py [hunk 1]
```diff
@@ -31,8 +31,8 @@ def ATG_formula(TG, A, exponent, min_case_count=200):
                    min_case_count)
 
 
-class EvaluatedFormula():
-    """Class containing an evaluated formula.
+class Curve():
```
Comment (miskosz) [inline covid_graphs/covid_graphs/formula.py:34]: Naming is hard indeed :D

Comment (miskosz) [reply covid_graphs/covid_graphs/formula.py:34]: `Trace` would be an option.

File: covid_graphs/covid_graphs/country_graph.py [hunk 2]
```diff
@@ -219,5 +217,7 @@ def create_dashboard(data_dir: Path, server: Flask, graph_type: GraphType=GraphT
     type=str,
 )
 def show_country_plot(data_dir: Path, country_name: str):
-    country_graph = CountryGraph(data_dir=data_dir, country_name=country_name, graph_type=GraphType.Normal)
+    country_graph = CountryGraph(data_dir=data_dir,
+                                 country_name=country_name,
+                                 graph_type=GraphType.Normal)
```
Comment (miskosz) [inline covid_graphs/covid_graphs/country_graph.py:222]: Btw. I plan to add to CI a style checker over the weekend, together with type checker and a testing framework.

**Review by miskosz — APPROVED** [review]
LGTM!

Food for thought (not this PR): From my experience, classes are used in python sparsely - only when the object needs to persist some state. In general, pure functions are preferred, placing them in the right submodule (=file) manages the logical grouping. This helps keeping the code modular and also easy to test.

In the case of `Curve`, I would instead create a dataclass with attributes `t, y, date_list` etc., and even add `first_idx` and `last_idx`. Then `create_curves` can be a pure function that returns a `Curve`.

File: covid_graphs/covid_graphs/country_graph.py (hunk 2 above)
Comment (miskosz) [reply covid_graphs/covid_graphs/country_graph.py:222]: Actually I meant automatic code formatter :)

**Comment by lukipuki** [comment]
Yeah `create_curves` is a static method, so the class is really just a simple data holder. I'm not familiar with dataclasses yet (one step at a time!).

**Pull request merged**